src/reducers/computeRenderedRows.js

```diff
@@ -101,6 +111,7 @@ function calculateRenderedRowRange(state, scrollAnchor) {
       endBufferIdx: 0,
```

Comment:
the early return here meant that we didn't assign to the state... so instead of assigning these to the state here and down below, I moved them both back to the caller.